src/app/dataset/tabheader/tabheaders.component.html

```diff
@@ -31,10 +33,12 @@
   <li [ngClass]="{ active: activeTab === 'log', disabled: disabled }">
     <a routerLink="/dataset/log/{{ datasetId }}">
       <span class="tab-title">{{ 'processingHistory' | translate }}</span>
-      <span class="tab-subtitle" *ngIf="lastExecutionData">{{ 'lastModified' | translate }}</span>
-      <span class="tab-subtitle tab-date" *ngIf="lastExecutionData">{{
-        lastExecutionData.updatedDate | date: 'dd/MM/yyyy - HH:mm'
-      }}</span>
+      <ng-container *ngIf="lastExecutionData">
+        <span class="tab-subtitle">{{ 'lastModified' | translate }}</span>
+        <span class="tab-subtitle tab-date" *ngIf="lastExecutionData">{{
```

Comment:
Duplicate *ngIf statement.

Reply:
The point of the grouping was to avoid this - you're paying attention!